README.md

```diff
@@ -38,13 +38,16 @@ Following libraries need to be available:
 * [kiwix-lib](https://github.com/kiwix/kiwix-lib/)
 * [Qt](https://www.qt.io/)
 * [aria2](https://aria2.github.io/)
+* [vue.js](https://vuejs.org/js/vue.js)
 
 These dependencies may or may not be packaged by your operating
 system. They may also be packaged but only in an older version. The
 compilation script will tell you if one of them is missing or too old.
 In the worse case, you will have to download and compile bleeding edge
 version by hand.
 
+The [vue.js](https://vuejs.org/js/vue.js) file has to be moved in the `kiwix-desktop/resources/js` directory. This is done automatically if you use [kiwix-build](https://github.com/kiwix/kiwix-build).
```

Comment:
As said in kiwix/kiwix-build#442, `kiwix-desktop` must not depends of `kiwix-build`.
`kiwix-build` is a "simple" tool that install dependencies before launching specific project build system. It could be replace by anything else (and this is the case for flatpak and windows).